Makefile

```diff
@@ -32,8 +32,8 @@ distclean: clean
 download/leaflet-$(DEP_LEAFLET_VERSION).tar.gz:
 	mkdir -p download
 	# do this in 2 steps to make sure only a completely downloaded file is used
-	wget -O download/leaflet.tar.gz https://github.com/Leaflet/Leaflet/archive/v$(DEP_LEAFLET_VERSION).tar.gz
-	mv download/leaflet.tar.gz download/leaflet-$(DEP_LEAFLET_VERSION).tar.gz
+	wget -O download/leaflet.zip https://leafletjs-cdn.s3.amazonaws.com/content/leaflet/v$(DEP_LEAFLET_VERSION)/leaflet.zip
```

Comment:
The URL https://github.com/Leaflet/Leaflet/releases/download/v1.9.3/leaflet.zip also works by the way, so we can choose if the Github or Amazon CDN is preferred.